src/modules/frost/bench_impl.h

```diff
@@ -0,0 +1,221 @@
+/***********************************************************************
+ * Copyright (c) 2025 Bank of Italy                                    *
+ * Distributed under the MIT software license, see the accompanying    *
+ * file COPYING or https://www.opensource.org/licenses/mit-license.php.*
+ ***********************************************************************/
+
+#ifndef SECP256K1_MODULE_FROST_BENCH_H
+#define SECP256K1_MODULE_FROST_BENCH_H
+
+#include "../../../include/secp256k1_frost.h"
+#include "fill_random.h"
+
+#define MSGLEN_FROST_BENCH (32)
```

Comment:
A previous version of this code used `MSGLEN` (defined in schnorrsig) instead of a macro defined in this file. The effect was that the library did not compile when built with frost enabled but schnorrsig disabled.

The current version defines `MSGLEN_FROST_BENCH` in order to solve this problem.